source/docs/controls-overviews/control-system-hardware.rst

```diff
@@ -25,23 +25,41 @@ NI roboRIO
 
-Voltage Regulator Module
-------------------------
+REV Power Distribution Hub
+--------------------------
+
+.. image:: images/control-system-hardware/power-distribution-hub.png
+  :alt: REV Power Distribution Hub
+  :width: 500
+
+The `REV Power Distribution Hub <https://docs.revrobotics.com/rev-11-1850/>`__ (PDH) is designed to distribute power from a 12VDC battery to various robot components. With 20 high-current (40A max) channels, 3 low-current (15A max), and 1 switchable low-current channel, the PDH gives teams more flexibility for overall power delivery. The Power Distribution Hub features toolless latching WAGO terminals, an LED voltage display, and the ability to connect over CAN or USB-C to the REV Hardware Client for real-time telemetry, making it easier than ever to wire and debug your robot.
+
+CTRE Voltage Regulator Module
+-----------------------------
 
 .. image:: images/control-system-hardware/voltage-regulator-module.png
-  :alt: Voltage Regulator Module
+  :alt: CTRE Voltage Regulator Module
   :width: 500
 
-The Voltage Regulator Module (VRM) is an independent module that is powered by 12 volts. The device is wired to a dedicated connector on the PDP. The module has multiple regulated 12V and 5V outputs. The purpose of the VRM is to provide regulated power for the robot radio, custom circuits, and IP vision cameras. For more information, see the `VRM User Manual <https://www.ctr-electronics.com/VRM%20User's%20Guide.pdf>`__.
+The CTRE Voltage Regulator Module (VRM) is an independent module that is powered by 12 volts. The device is wired to a dedicated connector on the PDP. The module has multiple regulated 12V and 5V outputs. The purpose of the VRM is to provide regulated power for the robot radio, custom circuits, and IP vision cameras. For more information, see the `VRM User Manual <https://www.ctr-electronics.com/VRM%20User's%20Guide.pdf>`__.
+
+REV Radio Power Module
+----------------------
+
+.. image:: images/control-system-hardware/radio-power-module.png
+  :alt: REV Radio Power Module
+  :width: 500
+
+The `REV Radio Power Module <https://docs.revrobotics.com/rev-11-1856/>`__ is designed to keep one of the most critical system components, the OpenMesh OM5P-AC WiFi radio, powered in the toughest moments of the competition. Traditional barrel jacks easily work themselves loose and often require hacks like hot glue to prevent intermittent power losses. The Radio Power Module eliminates the need for powering the radio through a traditional barrel power jack. Utilizing 18V Passive POE with two socketed RJ45 connectors, the Radio Power Module passes signal between the radio and roboRIO while providing power directly to the radio. After connecting the radio and roboRIO, easily add power to the Radio Power Module by wiring it to the low-current channels on the Power Distribution Hub utilizing the color coded push button WAGO terminals.
```

Comment:
Remove the part number of the radio since there's two legal open mesh PNs.

Reply:
The part about barrel jacks sounds like marketing and not documentation. Recommend removing.